compiler/q-implant/README.md

```diff
@@ -2,13 +2,21 @@
 
 _q-implant_ writes quantization parameters and weights (given as .json and .npy files) to a circle model.
 
+Therefore, before proceeding with q-implant, you should define `qparam.json` and `\*.npy` files so that they can be applied to the same operand as the circle you want to apply.
```

Comment:
Please remove the sentence and add the below content

```suggestion
```

---

## Format of input files (`.json` and `.npy`)

The main input file is a `.json` file, which is a dictionary whose key is a tensor name and value is quantization parameters and values (if exists).

```
{
  <tensor_name>: {
    "dtype": <dtype>,
    "scale": <path/to/scale_npy>,
    "zerop": <path/to/zerop_npy>,
    "quantized_dimension": <dim>,
    "value": <path/to/value_npy>
  },
  ...
}
```

`<tensor_name>`: String (target tensor name)
`<dtype>`: String (data type of the target tensor. ex: "uint8" or "int16")
`<path/to/scale_npy>`: String (path to the .npy file that contains scale. The npy file has to be 1d array of fp32.)
`<path/to/zerop_npy>`: String (path to the .npy file that contains zerop. The npy file has to be 1d array of int64.)
`<dim>`: Integer (quantized dimension)
`<path/to/value_npy>`: String (path to the .npy file that contains zerop. The npy file should have the same shape/dtype with the target tensor.)

NOTE "value" is an optional attribute. It is only necessary for weights.
NOTE `.npy` files have to be placed in the same directory with `qparam.json`.

Reply:
would be good to add something like: `.npy is numpy saved blabla ... file ...`